charts/newrelic-infrastructure/templates/ksm/deployment.yaml

```diff
@@ -49,11 +49,11 @@ spec:
         {{- . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "newrelic.common.serviceAccount.name" . }}
-      hostNetwork: {{ include "newrelic.common.hostNetwork.value" . }}
+      hostNetwork: {{ include "nriKubernetes.ksm.hostNetwork.value" . }}
       {{- if include "nriKubernetes.controlPlane.hostNetwork" . }}
       dnsPolicy: ClusterFirstWithHostNet
       {{- end }}
```

Comment:
This is a bit out of the scope of this PR, but a known bug is that `dnsPolicy` is set depending on `nriKubernetes.controlPlane.hostNetwork`, not `nriKubernetes.ksm.hostNetwork`. Do you think you could change that as well?

Reply:
Done